apps/wallet-mobile/src/features/AddWallet/common/CardAboutPhrase/CardAboutPhrase.tsx

```diff
@@ -0,0 +1,84 @@
+import {useTheme} from '@yoroi/theme'
+import React from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import {Space} from '../../../../components/Space/Space'
+
+type CardAboutPhraseProps = {
+  items: string[]
+  title?: string
+  background?: boolean
+  padding?: boolean
```

Comment:
`includeSpacing` padding is the implementation, tomorrow it can be a margin how knows, avoid leaking the implementation.